scripts/envs/chatbot

```diff
@@ -4,16 +4,7 @@ export APP_DESC="Chatbot Application example with LLM enabled chat applications"
 export APP_TAGS='["ai", "llamacpp", "vllm", "python"]'
 export APP_RUN_COMMAND="streamlit run chatbot_ui.py"
 export INIT_CONTAINER="quay.io/redhat-ai-dev/granite-7b-lab:latest"
```

Comment:
for all the env files, can we have a comment at the top? indicating the base env value is under `./env` and can be override in this file?

Reply:
@yangcao77 I think is a good idea. I've added a comment in every env file.